esx_service/utils/vmdk_utils.py

```diff
@@ -50,19 +55,23 @@ def get_datastores():
 def get_volumes():
     """ Return dicts of docker volumes, their datastore and their paths """
     volumes = []
-    for (datastore, path) in get_datastores():
+    for (datastore, url_name, path) in get_datastores():
         for file_name in list_vmdks(path):
             volumes.append({'path': path,
                             'filename': file_name,
                             'datastore': datastore})
     return volumes
 
 
+def get_vmdk_name(path, vol_name):
+    """ forms full name as <path-to-volumes>/<volname>.vmdk"""
+    return os.path.join(path, "%s.vmdk" % vol_name)
```

Comment:
Nit: I've been trying to stick to the more modern python recommendation of using `"{0}.vmdk".format(vol_name)`, but it doesn't really matter.